src/services/video_service.py

```diff
@@ -1,6 +1,66 @@
-from src.classes.video_data import VideoData
+# MIT License
+#
+# Copyright (c) 2023 Victor Calderon and Travis Craft
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
 
+import logging
 
-def download_video(url: str) -> VideoData:
-    print(f"Downloading video from the following URL: {url}...")
-    return VideoData(url=url, raw_video=[])
+from src.classes.video_data import VideoDataDAO
+
+__author__ = ["Victor Calderon and Travis Craft"]
+__maintainer__ = ["Victor Calderon and Travis Craft"]
+__all__ = []
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+logger.setLevel(logging.INFO)
+
+
+class VideoService(object):
+    """
+    Class object for the Video service.
+    """
+
+    def __init__(self, video_dao: "VideoDataDAO") -> None:
+        """
+        Class object for the Video Service.
+        """
+        self.video_dao = video_dao
+
+    def process_video(self):
+        """
+        Method for processing the input video.
+        """
+
+        return
+
+
+def download_video(url: str) -> VideoDataDAO:
```

Comment:
I feel like we may make this a method on the `VideoService` long-term. We'll see how everything develops.